getting-started/recipe-development-environment.md

```diff
@@ -54,6 +54,8 @@ dependencies {
 
     // rewrite-java dependencies only necessary for Java Recipe development
     implementation("org.openrewrite:rewrite-java")
+
+    // add based on your Java needs
```

Comment:
I wanted to clarify that you don't need all of these for a given project.

Reply:
Technically all that's needed is the one corresponding to the JDK version you'll be running on. They can coexist on a classpath. So it's a bit redundant to have all three, but it's also maximally flexible for whatever JDK version they end up using.

Reply:
Technically all that's needed is the one corresponding to the JDK version you'll be running on. They can coexist on a classpath. So it's a bit redundant to have all three, but it's also maximally flexible for whatever JDK version they end up using. There's no need for users to spend effort considering this point if they accept the default. If we're going to have a comment explaining, which is fine, it should probably not imply that they need to consider and make a choice when no such consideration or choice is necessary.